examples/index.html

```diff
@@ -164,6 +171,12 @@
                 loadManifest($(this).data('json'));
             });
 
+            $('#loadIIIF').click(function () {
+            var jsonUrl = $('#inputIIIF').val();
+            if (jsonUrl) {
+                loadManifest(jsonUrl);
+                }
+            });

```

Comment:
This looks good to me, except I think the indentation here got a little out of whack.

```suggestion
                var jsonUrl = $('#inputIIIF').val();
                if (jsonUrl) {
                    loadManifest(jsonUrl);
                }
            });

```